buildenv/jenkins/common/pipeline-functions

```diff
@@ -406,14 +406,29 @@ def get_test_target_names() {
     return targetNames
 }
 
-def get_build_job_name(spec, version) {
-    return "Build-JDK${version}-${spec}"
+def get_build_job_name(spec, version, identifier) {
+    id = convert_build_identifier(identifier)
+    return "Build-JDK${version}-${spec}-${id}"
 }
 
-def get_test_job_name(targetName, spec, version) {
-    return "Test-${targetName}-JDK${version}-${spec}"
+def get_test_job_name(targetName, spec, version, identifier) {
+    id = convert_build_identifier(identifier)
+    return "Test-${targetName}-JDK${version}-${spec}-${id}"
```

Comment:
- Please see SPEC to ARCH_OS map in https://github.com/eclipse/openj9/issues/5210#issuecomment-478599394
- test job name format should be `Test_openjdk${version}_j9_${targetName}_${ARCH_OS}_${id}`